modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -114,7 +114,20 @@ org.mule.runtime.core.internal.message.DefaultMessageBuilder$MessageImplementati
 }
 ----
 +
-* `#[error.childErrors]` returns: `[]`
+Note that you can return the payload or attributes (metadata)
+associated with the error message by using the following constructions:
+`&#35;[error.errorMessage.payload]`, `&#35;[error.errorMessage.attributes]`.
+Depending on the metadata supported by a given connector, you can
+also select values of fields for attributes, such as the status code from
+an HTTP request (for example, `&#35;[error.errorMessage.attributes.statusCode]`).
+However, because `statusCode` is a runtime field that is not available at
```

Comment:
I would not explain this to users. I think it's too specific. If you think mentioning this is worth it, then I would make it generic and say that the metadata support in Studio is limited because of how dynamic error handling can be.

Reply:
ok, I'll DM you. Trying to address the issue in the SE.

Reply:
generalized based on discussion